test/elements/depayloader_test.exs

```diff
@@ -68,20 +68,21 @@ defmodule DepayloaderTest do
 
     assert_end_of_stream(pipeline, :sink)
     refute_sink_buffer(pipeline, :sink, _, 2000)
+    Pipeline.terminate(pipeline, blocking?: true)
```

Comment:
in Core v0.12 there is no more `blocking?` option - the termination is "blocking" by default, and you can make it non-blocking with `asynchronous?: true` option.